src/client/content_cao.cpp

```diff
@@ -1580,6 +1607,8 @@ void GenericCAO::processMessage(const std::string &data)
 	u8 cmd = readU8(is);
 	if (cmd == AO_CMD_SET_PROPERTIES) {
 		ObjectProperties newprops;
+		newprops.show_on_minimap = m_is_player; // default
```

Comment:
`!m_is_player` would be more correct, though this variable should default to `true` by default if there's no such property in `is` - also because the local player case is already handled below.

Reply:
Huh? Normal entities are not supposed to show on the minimap by default.

Reply:
Sorry, I somehow assumed `m_is_player` is the same as `m_is_local_player`...